lib/modules/manager/pipenv/extract.spec.ts

```diff
@@ -99,7 +114,7 @@ describe('modules/manager/pipenv/extract', () => {
           },
         ],
         extractedConstraints: {
-          python: '== 3.6.*',
+          python: '== 3.6.2',
```

Comment:
why this change? 🤔

Reply:
This is how artifact extraction logic works, it turned out to be different than `extract.ts`